src/Schemas/Model/Attributes/Attribute.php

```diff
@@ -214,12 +231,16 @@ public static function fromArray(string $name, array $data): self
         $name = $data['name'] ?? $name;
         $label = $data['label'] ?? '';
         $type = Type::from($data['type'] ?? 'string');
-        $attribute = new self($name, $label, $type);
+        $aliasKey = $data['aliasKey'] ?? false;
+        $attribute = new self($name, $label, $type, $aliasKey);
```

Comment:
@ShreyDhyani alias need to be removed from self, You are already setting it below

Reply:
Removed